client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss

```diff
@@ -30,9 +34,12 @@
 		background-attachment: fixed;
 		background-size: auto 15%, auto 15%;
 
-		@include break-large {
+		@include break-xlarge {
```

Comment:
Let's go break-huge instead. `x-large` is only 1080 and these images are huge.

Reply:
Cool. Next commit will address it! Thank you for checking.